README.md

```diff
@@ -7,7 +7,7 @@ Code for the tutorials found on the
 
 ## Install
 
-Note: [NodeJS](https://nodejs.org/en/download/) (v12+) must be installed to run
+Note: [NodeJS](https://nodejs.org/en/download/) (v16+) must be installed to run
```

Comment:
Minimum Node.JS support is v18 now since v25

Reply:
Technically v16 still works fine (it's what I've been using). But updated to 18 since it is LTS.